src/components/Table/GridTable.test.tsx

```diff
@@ -2110,6 +2111,19 @@ describe("GridTable", () => {
     expect(cell(r, 1, 1)).toHaveStyleRule("background-color", Palette.LightBlue50);
   });
 
+  it("reacts to setting activeCellId", async () => {
+    // Given a table initially rendered with an active cell id
+    const r = await render(
+      <GridTable columns={columns} rows={rows} style={{ cellHighlight: true }} activeCellId={"data_1_name"} />,
+    );
+    // Then expect the cell to have highlight style
+    expect(cell(r, 1, 0)).toHaveStyleRule("border-width", "1px");
+    expect(cell(r, 1, 0)).toHaveStyleRule("border-color", Palette.LightBlue900);
+    // Then expect other cells not to have highlight style
+    expect(cell(r, 1, 1)).not.toHaveStyleRule("border-width", "1px");
+    expect(cell(r, 1, 1)).not.toHaveStyleRule("border-color", Palette.LightBlue900);
+  });
```

Comment:
I really like the test for the style rules. Can we write tests to ensure (1) The active cell is updated if the prop value changes, and (2) the active cell can be changed via the GridTableApi?